src/compiler/config/outputs/validate-dist.ts

```diff
@@ -135,13 +138,13 @@ const validateOutputTargetDist = (config: d.ValidatedConfig, o: d.OutputTargetDi
     typesDir: o.typesDir || DEFAULT_TYPES_DIR,
     esmLoaderPath: o.esmLoaderPath || DEFAULT_ESM_LOADER_DIR,
     copy: validateCopy(o.copy ?? [], []),
-    polyfills: isBoolean(o.polyfills) ? o.polyfills : undefined,
+    polyfills: typeof o.polyfills === 'boolean' ? o.polyfills : false,
```

Comment:
Just curious, any reason to move away from `isBoolean()`?

Reply:
just doesn't seem necessary to me, and I noticed recently that the `isBoolean` helper in node `utils` is marked as deprecated

Reply:
I think this is fine, the only thing I'm confused by is:
> I noticed recently that the isBoolean helper in node utils is marked as deprecated

Doesn't `isBoolean` resolve to https://github.com/ionic-team/stencil/blob/cb3eac3a2b7795a5c8d3f9100812337d1a8d8a9d/src/utils/helpers.ts#L171?

Reply:
yeah it does, I just referenced the node utils deprecation because the reason for it was basically "why not just use `typeof` directly" which I think applies to our little helper too

Reply:
I think the "deprecation" is my source of confusion - to clarify, it's not marked as deprecated in the code, right? Did we have a conversation about deprecating this as a team? TBH, I don't quite remember 🤔

Reply:
sorry should have been more clear, I'm referring to this function: https://github.com/nodejs/node/blob/main/doc/api/util.md#utilisbooleanobject

Reply:
Hmm, don't really understand why they deprecated their util. It doesn't make any difference technically, but feels like we should be consistent by either:
1. Continuing to use our util **or**
2. Deprecating our util

I don't think Node deprecating theirs needs to influence us here, but I don't really care which way we go either. Not gonna consider this a blocker for me either.

Reply:
I think it's just a little bit more direct what's going on, like there's a keyword in the language which does what's needed here, so there's no reason to have a util function which doesn't do anything other than return the result of a comparison based on that keyword

that said, this seems controversial and I'm not at all committed to it so I will remove